### PR TITLE
Inventory - Fix Rule Asset Import Case

### PR DESCRIPTION
**Pull request by tsmr**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #none

When you want to Link a agent on an existing item or no import agent like this

image

The result is not send to $output without this check

### CODE REVIEW

**Review by orthagh — CHANGES_REQUESTED** [review]
To note, a bit bellow , there is an `else` for triggering the passing rule.
This should check the chosen action can import the asset:

Something like that
image

**Review by orthagh — APPROVED** [review]

**Comment by tsmr** [comment]
Yes but it's not a new equipement. It's an equipment to link to another (send not same result)

**Review by cedric-anne — APPROVED** [review]

File: src/RuleImportAsset.php [hunk 1]
```diff
@@ -906,7 +907,7 @@ function executeActions($output, $params, array $input = []) {
                      }
                      return $output;
                   }
-               } else {
+               } elseif ($action->fields["value"] != self::RULE_ACTION_LINK_OR_NO_IMPORT) {
```
Comment (cedric-anne) [inline src/RuleImportAsset.php:910]:
```suggestion
               } else if ($action->fields["value"] != self::RULE_ACTION_LINK_OR_NO_IMPORT) {
```

Comment (trasher) [reply src/RuleImportAsset.php]: :+1: thank you

**Pull request merged**